finetuner/experiment.py

```diff
@@ -139,6 +148,48 @@ def create_run(
             if isinstance(callback, EvaluationCallback):
                 eval_callback = callback
 
+        if isinstance(train_data, (TextIO)) or (
+            isinstance(train_data, str) and isfile(train_data)
+        ):
+            model_stub = get_stub(model, select_model='clip-text')
+            if csv_options:
+                train_data = DocumentArray(
+                    from_csv(
+                        file=train_data,
+                        task=model_stub.task,
+                        size=csv_options.get('size', None),
```

Comment:
lets not accept an arbitrary dict for CSV options. You can create a dataclass for it

Reply:
also document what are the csv options